lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -205,6 +205,36 @@ _.assign(Builders.prototype, {
         return this.options.retainEmptyValues ? null : undefined;
     },
 
+    /**
+     * Creates options for body from v2 format.
+     *
+     * @param {Object} item - The v2 request to derive information form.
+     * @returns {Object} - The converted body options.
+     */
+    dataOptions: function (item) {
+        if (item.request && item.request.body) {
+            let bodyOptions = item.request.body.options;
```

Comment:
```suggestion
            var bodyOptions = item.request.body.options;
```
still supports Node v4.

And, use `_.get` here.